### PR TITLE
[LPTOCPCI-469] Allow specifying multiple components

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

**Comment by smatula** [comment]
Tested against issues.stage
Settings:
export JOB_NAME_SAFE=servicemesh-interop-aws
export BUILD_ID=1678819350066237440
export JOB_NAME=periodic-ci-maistra-maistra-test-tool-main-servicemesh-ocp4.14-lp-interop-servicemesh-interop-aws
export FIREWATCH_CONFIG='[ {"step": "servicemesh-kiali-cypress-tests-execute", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "ENTESB", "jira_component": "Fuse Online,QE" } ]'
export FIREWATCH_DEFAULT_JIRA_PROJECT=WINDUP

Tickets https://issues.stage.redhat.com/browse/ENTESB-21425
[12:27](https://redhat-internal.slack.com/archives/D02BGFGDDGT/p1689611226886949)
Anything created from 21418 to 21425 was created by the tool...

**Review by ascerra — APPROVED** [review]

File: cli/objects/jira_base.py [hunk 1]
```diff
@@ -96,8 +96,13 @@ def create_issue(
             issue_dict.update({"labels": labels})  # type: ignore
 
         if component:
+            components = []
+            for comp in filter(None, component.split(",")):
+                if comp.strip():
+                    components.append(dict(name=comp.strip()))
```
[the remark this answers is not in the document]
Comment (ryankwilliams) [reply cli/objects/jira_base.py]: Thanks @smatula!

**Pull request merged**